src/energyplus/ForwardTranslator.cpp

```diff
@@ -4361,13 +4377,16 @@ namespace energyplus {
     }
   }
 
-  void ForwardTranslator::createStandardOutputRequests() {
+  void ForwardTranslator::createStandardOutputRequests(const model::Model& model) {
     if (!m_excludeHTMLOutputReport) {
-      IdfObject tableStyle(IddObjectType::OutputControl_Table_Style);
-      m_idfObjects.push_back(tableStyle);
-      tableStyle.setString(OutputControl_Table_StyleFields::ColumnSeparator, "HTML");
-      if (m_ipTabularOutput) {
-        tableStyle.setString(OutputControl_Table_StyleFields::UnitConversion, "InchPound");
+      boost::optional<model::OutputControlTableStyle> _tableStyle = model.getOptionalUniqueModelObject<model::OutputControlTableStyle>();
```

Comment:
(No changes needed, but FYI if we end up making our IDD (or Ctor) use the same defaults as the FT, you could just call model.getUniqueModelObject<XXX>() to instantiate the object as we operate on a copy of our model to begin with. This is maybe clearer this way though.)